### ISSUE TITLE
Technical Review #1 - Front End Functionality

### ISSUE THREAD

**Issue by Robert3631** [jwannamaker/neural-web-tool#70]
# Formal Technical Review (FTR) Report

## 1. Review Summary
* **Product under review:** Front-end implementation
* **Review Date:** April 25, 2026

## 2. Review Checklist & Findings
* **Syntactic Correctness:** Compiles, does not pass flint
* **Logic Errors:** Progress bar of the UI distorts the UI of the local div
* **Standards Compliance:** Doesn't follow Flint
* **Maintainability:** Yes, it code has comments

## 3. Review Summary Report (Meeting Minutes)
**Summary of Issues:**
1. Only the cross entropy loss function works
2. Progress bar does not fully load, and only goes about 2/3rds of the way
3. Inconsistencies with the server, occasionally goes down
4. Marking right or wrong on prediction feedback is slow to respond
5. Table moves when clicked on
6. Start training is spammable and breaks website
